scripts/build/dependencies-npm.sh

```diff
@@ -101,7 +101,7 @@ function run_install() {
   # since compiled add-ons will not work otherwise.
   npm rebuild
 
-  npm install $INSTALL_OPTS
+  npm install $INSTALL_OPTS --fetch-retries 100 --fetch-retry-maxtimeout 600000
```

Comment:
💯  – shouldn't 10 be enough? If there's a network issue persisting over a longer period of time, I think it'd make more sense to have the build fail quicker, rather than retry forever, to prevent the queue from clogging up.